specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2025-06-01-preview/common.json

```diff
@@ -0,0 +1,312 @@
+      "properties": {
+        "username": {
+          "type": "string",
+          "description": "Login username."
+        },
+        "password": {
+          "type": "string",
+          "description": "Login password.",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ],
+          "x-ms-secret": true
+        }
+      }
+    },
+    "ConnectionAuth": {
+      "type": "string",
+      "description": "The endpoint connection authentication type(s).",
+      "enum": [
+        "Windows_NTLM",
+        "Windows_Kerberos",
+        "Windows_Negotiate",
+        "Certificate",
+        "Windows_NTLM_Certificate",
+        "Windows_Kerberos_Certificate",
+        "Windows_Negotiate_Certificate",
+        "Certificate_Windows_NTLM",
+        "Certificate_Windows_Kerberos",
+        "Certificate_Windows_Negotiate"
+      ],
+      "x-ms-enum": {
+        "name": "connectionAuth",
+        "modelAsString": true
+      }
+    },
+    "EncryptionAlgorithm": {
+      "type": "string",
+      "description": "The encryption algorithm(s) used by the endpoint.",
+      "enum": [
+        "NONE",
+        "RC4",
+        "AES",
+        "NONE, RC4",
+        "NONE, AES",
+        "RC4, AES",
+        "AES, RC4",
+        "NONE, RC4, AES",
+        "NONE, AES, RC4"
+      ],
+      "x-ms-enum": {
+        "name": "encryptionAlgorithm",
+        "modelAsString": true
+      }
+    },
+    "SkuRecommendationResultsRecommendationStatus": {
+      "description": "The target recommendation Status for this database.",
+      "type": "string",
+      "enum": [
+        "NotReady",
+        "Ready",
+        "ReadyWithConditions",
+        "Unknown"
+      ],
+      "x-ms-enum": {
+        "name": "recommendationStatus",
+        "modelAsString": true
+      }
+    },
+    "SkuRecommendationResultsMonthlyCost": {
+      "description": "The monthly cost of the particular SKU.",
+      "type": "object",
+      "properties": {
+        "computeCost": {
+          "format": "float",
+          "description": "Represents the Cost of Compute.",
+          "type": "number"
+        },
+        "storageCost": {
+          "format": "float",
+          "description": "Represents the Cost of Storage.",
+          "type": "number"
+        },
+        "iopsCost": {
+          "format": "float",
+          "description": "Represents the Cost of IOPS.",
+          "type": "number"
+        },
+        "sqlLicenseCost": {
+          "format": "float",
+          "description": "Represents the Cost of SQL license.",
+          "type": "number"
+        },
+        "windowsLicenseCost": {
+          "format": "float",
+          "description": "Represents the Cost of Windows license.",
+          "type": "number"
+        },
+        "totalCost": {
+          "format": "float",
+          "description": "Represents the Total Cost.",
+          "type": "number"
+        }
```

Comment:
what are the units of the cost ? USD ? 

please update descriptions, and naming too - like computeCostInUSD etc.